crane/cmd.go

```diff
@@ -252,6 +268,11 @@ See the corresponding docker commands for more information.`,
 	cmdLogs.Flags().StringVarP(&options.tail, "tail", "", "all", "Output the specified number of lines at the end of logs")
 	cmdLogs.Flags().BoolVarP(&options.colorize, "colorize", "z", false, "Output the lines with one color per container")
 
+	cmdTmpl.Flags().StringVarP(&options.tmplInput, "template", "t", "", "The input template file name")
```

Comment:
I see your point. Now I'm not sure which name should have the feature… With `template` I understand better what it is intend, and the other ones are more ambiguous. But the action itself is not template but the file used to generate an output. So I would prefer `generate` for this point. The problem I see for using `generate` is that it's a very generic word.